presto-docs/src/main/sphinx/functions/string.rst

```diff
@@ -141,6 +141,10 @@ String Functions
     Returns the starting position of the first instance of ``substring`` in
     ``string``. Positions start with ``1``. If not found, ``0`` is returned.
 
+.. function:: starts_with(string, substring) -> boolean
+
+    Returns boolean whether ``string`` starts with ``substring`` or not
```

Comment:
How about "Tests whether ``substring`` is a prefix of ``string``".

@mosabua, any suggestions?

Reply:
thats good.. or